app/com/linkedin/drelephant/exceptions/core/ExceptionFingerprintingRunner.java

```diff
@@ -71,4 +79,40 @@ public void run() {
     logger.info("Total time spent in exception fingerprinting in  " + _analyticJob.getAppId() + " "
         + (endTime - startTime) * 1.0 / (1000000000.0) + "s");
   }
+
+  private void saveDriverExceptionLogForExceptionFingerPrinting(List<ExceptionInfo> exceptionInfoList) {
+    if (exceptionInfoList != null) {
+      final String NOT_APPLICABLE = "NA";
+      Optional<ExceptionInfo> driverExceptionInfo = exceptionInfoList.parallelStream()
+          .filter(ex -> ex.getExceptionSource().equals(ExceptionInfo.ExceptionSource.DRIVER))
+          .findFirst();
+      if (driverExceptionInfo.isPresent()) {
```

Comment:
If its false ,then what are you planning to do ?

Reply:
Will add the part to show some message on non-availablity of driver logs.